src/sections/Meshmap/meshmap.style.js

```diff
@@ -2,7 +2,11 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    color: #00d3a9;
```

Comment:
Please make reference to theme color (using variable).

Reply:
Lee, what do you mean by using variable.
something like this:
 --white: #ffffff;
color: var(--white)